plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.properties

```diff
@@ -3,4 +3,4 @@ providerName=Google, Inc.
 
 deployStandardCommandDescription=Deploy the project to App Engine Standard
 deployStandardCommandName=Deploy to App Engine Standard
```

Comment:
Looking at this now, why do we have a command name and a menu label? Are we using this in two different UI items?